src/Microsoft.IdentityModel.Tokens/Validators.cs

```diff
@@ -38,6 +38,36 @@ namespace Microsoft.IdentityModel.Tokens
     /// </summary>
     public static class Validators
     {
+        /// <summary>
+        /// Validates if a given algorithm for a <see cref="SecurityKey"/> is valid.
+        /// </summary>
+        /// <param name="algorithm">The algorithm being considered if it is valid</param>
+        /// <param name="securityKey">The <see cref="SecurityKey"/> that signed the <see cref="SecurityToken"/>.</param>
+        /// <param name="securityToken">The <see cref="SecurityToken"/> being validated.</param>
+        /// <param name="validationParameters"><see cref="TokenValidationParameters"/> required for validation.</param>
+        /// <returns><c>true</c> if the algorithm is acceptable.</returns>
+        public static void ValidateAlgorithms(string algorithm, SecurityKey securityKey, SecurityToken securityToken, TokenValidationParameters validationParameters)
```

Comment:
I think it may make more sense to call this "ValidateAlgorithm" (singular).

Reply:
Good idea. It is only checking one